packages/aws-cdk/test/integ/cli/test-cdk-version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Also, this file needs to be executable in order to run. It currently doesn't run.